fmriprep/cli/run.py

```diff
@@ -697,48 +697,41 @@ def build_workflow(opts, retval):
 
 
 def parse_spaces(opts):
-    """Ensures the spaces are correctly parsed"""
-    from sys import stderr
+    """
+    Ensures user defined output spaces are correctly parsed.
+
+    Certain options require normalization to a space not explicitly defined by users.
+    These spaces are labeled with a unique key ``no-output`` set to ``True``,
+    and will not be included in the final outputs.
+    """
     from collections import OrderedDict
     # Set the default template to 'MNI152NLin2009cAsym'
-    output_spaces = opts.output_spaces or OrderedDict([('MNI152NLin2009cAsym', {})])
+    target_spaces = opts.output_spaces or OrderedDict(
+        [('MNI152NLin2009cAsym', {'no-output': True})]
```

Comment:
I'm still a bit iffy on this change, so would like to draw attention here for any future reviews